test/e2e/scenario/recorder/shadowDom.scenario.ts

```diff
@@ -244,6 +281,29 @@ describe('recorder with shadow DOM', () => {
         ],
       })
     })
+
+  createTest('can record scroll from inside the shadow root')
+    .withRum({})
+    .withSetup(bundleSetup)
+    .withBody(html`
+      ${scrollableDivShadowDom}
+      <my-scrollable-div id="host" />
+    `)
+    .run(async ({ intakeRegistry }) => {
+      const div = await getNodeInsideShadowDom('my-scrollable-div', 'button')
+
+      await div.click()
```

Comment:
Not really, I just wanted to have a way to trigger the scroll once everything is setup (calling `scrollTo` on the element from the test isn't working because the returned `wdio.Element` doesn't allow this)

Reply:
Making the test itself trigger a scroll would be easier to follow but if it is not possible, adding a comment that clarify the intent could help

Reply:
✔️